src/youtube-player/youtube-player.ts

```diff
@@ -179,6 +183,16 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
   }
   private _playerVars = new BehaviorSubject<YT.PlayerVars | undefined>(undefined);
 
+  /** Whether cookies inside the player have been disabled. */
+  @Input()
+  get disableCookies(): boolean {
```

Comment:
I was on the fence between calling this `disableCookies` and having a `host` input where developers would pass in `https://www.youtube-nocookie.com` themselves. I went with the latter because it seems more ergonomic, but I'm open to changing it.